packages/x-charts/src/LineChart/AreaPlot.tsx

```diff
@@ -32,82 +99,29 @@ export interface AreaPlotProps
  * - [AreaPlot API](https://mui.com/x/api/charts/area-plot/)
  */
 function AreaPlot(props: AreaPlotProps) {
-  const { slots, slotProps, ...other } = props;
+  const { slots, slotProps, skipAnimation, ...other } = props;
 
-  const seriesData = React.useContext(SeriesContext).line;
-  const axisData = React.useContext(CartesianContext);
-
-  if (seriesData === undefined) {
-    return null;
-  }
-  const { series, stackingGroups } = seriesData;
-  const { xAxis, yAxis, xAxisIds, yAxisIds } = axisData;
-  const defaultXAxisId = xAxisIds[0];
-  const defaultYAxisId = yAxisIds[0];
+  const completedData = useCompletedData();
```

Comment:
Nitpick: it feels like the `completed` word infers that something is missing in the original data. 🙈 
Have you considered going for `useParsedData` or `useFormattedData` instead? 🤔

Reply:
I used `useAggregatedData` because it aggregates stacking info, plus scaling info into a single array of rectangle to plot

Reply:
Makes total sense. 👍